db/db_local/install.go

```diff
@@ -24,6 +25,18 @@ import (
 
 var ensureMux sync.Mutex
 
+func noBackupWarning() string {
+	warningMessage := `
```

Comment:
warningMessage := `the Steampipe database has been upgraded from Postgres 12 to Postgres 14. Data in the public schema has not been migrated.

If you want to restore the contents of your public schema, please contact Steampipe support.`